Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -227,22 +219,30 @@ public function build(array $buildSubject)
     }
 
     /**
-     * @param \Magento\Sales\Model\Order $order
+     * @param array $requestBody
+     * @param Order $order
      * @return array
      */
-    private function getOtherDeliveryInformation(\Magento\Sales\Model\Order $order): array
+    private function addOtherDeliveryInformation(array $requestBody, Order $order): array
```

Comment:
Can we keep this function to only require the order and return the array as before? I think it should be the responsibility of wherever this function is from to add this array to the body, not the responsibility of the function itself.

I think with this implementation the function is way too tightly bound to the location where we're calling it from.

Reply:
Thank you for your review @Morerice, I committed that suggestion.